apps/web-giddh/src/app/contact/contact.component.ts

```diff
@@ -409,12 +422,20 @@ export class ContactComponent implements OnInit, OnDestroy {
             this.searchStr$.next(searchedText);
         });
 
-        this.store.pipe(select(state => state.company && state.company.integratedBankList), takeUntil(this.destroyed$)).subscribe(response => {
-            if(response?.length > 0) {
+        this.store.pipe(select(state => state.company && state.company), takeUntil(this.destroyed$)).subscribe(response => {
```

Comment:
Please check this is getting removed, if the change is required then `state.company` is redundant within selector
```suggestion
        this.store.pipe(select(state => state.company && state.company.integratedBankList), takeUntil(this.destroyed$)).subscribe(response => {
```